### PR TITLE
AndresC-AuthMethodsPolicy-SMS

### PR DESCRIPTION
**Pull request by andres-canello**

### CODE REVIEW

**Review by mmcla — APPROVED** [review]

File: api-reference/beta/api/smsauthenticationmethodconfiguration-update.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  "name": "update_smsauthenticationmethodconfiguration"
+}
+-->
+``` http
+PATCH https://graph.microsoft.com/beta/policies/authenticationMethodsPolicy/authenticationMethodConfigurations/sms
+Content-Type: application/json
+Content-length: 100
+
+{
+  "@odata.type": "#microsoft.graph.smsAuthenticationMethodConfiguration",
```
Comment (davidmu1) [inline api-reference/beta/api/smsauthenticationmethodconfiguration-update.md:74]: These should be realistic looking values.

File: api-reference/beta/resources/smsauthenticationmethodtarget.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+A collection of users or groups enabled to use [Text Message authentication methods policy](../resources/smsAuthenticationMethodConfiguration.md) in Azure AD.
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|Object ID of an Azure AD user or group.|
+|isRegistrationRequired|Boolean|Determines whether the user is enforced to register the authentication method. *Not supported*.|
```
Comment (davidmu1) [inline api-reference/beta/resources/smsauthenticationmethodtarget.md:21]:
```suggestion
|isRegistrationRequired|Boolean|Determines whether the user is enforced to register the authentication method. **Not supported**.|
```

File: api-reference/beta/resources/smsauthenticationmethodtarget.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+|isUsableForSignIn|Boolean|Determines if the users or groups can use this authentication method to sign in to Azure AD. *Must always be True*|
```
Comment (davidmu1) [inline api-reference/beta/resources/smsauthenticationmethodtarget.md:22]:
```suggestion
|isUsableForSignIn|Boolean|Determines if the users or groups can use this authentication method to sign in to Azure AD. The value is always `true`.|
```

**Review by davidmu1 — APPROVED** [review]

File: api-reference/beta/api/smsauthenticationmethodconfiguration-update.md (hunk 1 above)
Comment (andres-canello) [reply api-reference/beta/api/smsauthenticationmethodconfiguration-update.md:74]: Done

File: api-reference/beta/resources/smsauthenticationmethodtarget.md (hunk 3 above)
Comment (andres-canello) [reply api-reference/beta/resources/smsauthenticationmethodtarget.md:22]: done

File: api-reference/beta/resources/smsauthenticationmethodtarget.md (hunk 2 above)
Comment (andres-canello) [reply api-reference/beta/resources/smsauthenticationmethodtarget.md:21]: done

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please add the new methods to the beta TOC file. Thanks!

**Comment by andres-canello** [comment]
> Please add the new methods to the beta TOC file. Thanks!

Thanks for catching that @Lauragra , done. Andres

**Review by Lauragra — APPROVED** [review]

**Pull request merged**